### PR TITLE
feat: add .gitattributes to unify different platforms eol

### PR DESCRIPTION
**Pull request by iyear**
## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](../CONTRIBUTING.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests

## Description
Maybe it's a breaking change.
**Windows users may have all files changed after pulling the latest code, so just delete all files except `.gitattributes` `.git` and roll them back to reset the eol.Or nothing may happen, just make sure to show the eol of all text files as `LF` in the IDE**

Add .gitattributes to unify different platforms eol

## Related Issues
#541 

## New Behavior (screenshots if needed)
After I run `make fmt` in `WSL`, no file changed.
image

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+
+*.{png,jpg,jpeg,gif,webp,woff,woff2} binary
```
Comment (IronCore864) [inline .gitattributes:3]: Missing "EOF" here, as suggested by the red sign. Maybe configure your IDE so that a new line at the end of a file is ensured.

**Review by IronCore864 — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (daniel-hutao) [reply .gitattributes]: /cc @iyear I've fixed it. And please remember to add new line at the end of any files next time.

Comment (daniel-hutao) [reply .gitattributes]: 054c85bd06d02ae290913706ce8ecbfc3834e5c8

**Review by daniel-hutao — APPROVED** [review]

**Comment by daniel-hutao** [comment]
close #541 

/cc @iyear Please see the document [here](https://docs.devart.com/studio-for-sql-server/source-controlling-databases/associating-commits-with-github-issues.html) about how to close a issue with comment. And remember close the issue with pr comment next time.

**Comment by iyear** [comment]
@daniel-hutao @IronCore864   Thanks for the review and help! My mistake, I will pay attention next time.

Related Links: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Pull request merged**